stm32cube/stm32h7xx/drivers/include/stm32h7xx_ll_adc.h

```diff
@@ -6238,10 +6238,14 @@ __STATIC_INLINE uint32_t LL_ADC_GetOverSamplingDiscont(ADC_TypeDef *ADCx)
   *         ADC state:
   *         ADC must be disabled or enabled without conversion on going
   *         on either groups regular or injected.
+  * @note   Caution: Oversampling Ratio is dependent to ADC instance and IP version:
+  *         For STM32H72x/3x: ADC3 has 3-bit OVSR, and 9-bit OVSR for ADC1/ADC2
+  *         For Rest of STM32H7xxx All ADCs have 9-bit OVSR.
   * @rmtoll CFGR2    OVSS           LL_ADC_ConfigOverSamplingRatioShift\n
   *         CFGR2    OVSR           LL_ADC_ConfigOverSamplingRatioShift
   * @param  ADCx ADC instance
-  * @param  Ratio This parameter can be in the range from 1 to 1024.
+  * @param  Ratio This parameter can be in the range from 1 or 1024
+  *               or 2, 4, 8, 16, 32, 64, 128, 256 for ADC3.
```

Comment:
This function should accept the 3-bit value, and not the 10-bit value. For example, it should be able to accept the define [LL_ADC_OVS_RATIO_2](https://github.com/STMicroelectronics/STM32CubeH7/blob/5975bffae9358bc2b2890a35a203d940a395efef/Drivers/STM32H7xx_HAL_Driver/Inc/stm32h7xx_ll_adc.h#L1397-L1412).

These defines seems to be shifted to the correct location already. E.g:
```
LL_ADC_OVS_RATIO_256 = 28 = 0b0011100
```
I understand that this is drivers only used by Zephyr, and that Zephyr might not use `LL_ADC_OVS_RATIO_2`, but the fix for this [#177](https://github.com/STMicroelectronics/STM32CubeH7/issues/177) should be able to accept `LL_ADC_OVS_RATIO_2`, which will create a conflict.

Reply:
If so, this is a change in the LL API
LL_ADC_ConfigOverSamplingRatioShift ratio is a uint32 value from 1 to 1024, including for ADC3 of the stm32h72x, the range is 2..256.